install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -52,8 +52,8 @@ ldap.example.com).
 +
 . Prepare the VMs:
 +
-* *_proxy.example.com_*: A VM to use as the authenticating proxy. This machine must
-have at least SSSD 1.12.0 available, which means a fairly recent operating
+* *_remote-basic.example.com_*: A VM to use as the remote basic authentication server.
+This machine must have at least SSSD 1.12.0 available, which means a fairly recent operating
 system. This topic uses a Red Hat Enterprise Linux 7.2 server for its examples.
 +
 * *_openshift.example.com_*: A VM to use to run {product-title}.
```

Comment:
Would a more realistic use-case be to set up the LDAP failover for an existing production cluster? (I don't see where this VM actually comes into play.)